src/sfctl/config.py

```diff
@@ -70,21 +70,22 @@ def set_ca_cert(ca_path=None):
     else:
         set_config_value('use_ca', 'false')
 
-def cert_info():
+def cert_info(security_type):
```

Comment:
So that it can skip an attempt to read the config again. Whoever calls this method should know which security type because the return would be different.